weaknesses/MASVS-CODE/MASWE-0116.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Compiler Provided Security Features Not Used
+id: MASWE-0116
+alias: compiler-provided-security-features-not-implemented
+platform: [android, ios]
+profiles: [L2]
+mappings:
+  masvs-v2: [MASVS-CODE-3, MASVS-CODE-4]
+refs:
+- https://cs.android.com/android/platform/superproject/main/+/main:bionic/linker/linker_main.cpp;l=397?q=linker_main&ss=android%2Fplatform%2Fsuperproject%2Fmain
+- https://www.mcafee.com/enterprise/en-us/assets/white-papers/wp-secure-coding-android-applications.pdf
```

Comment:
```suggestion
```
Link is dead, wayback machine still has it: https://web.archive.org/web/20210414040405/https://www.mcafee.com/enterprise/en-us/assets/white-papers/wp-secure-coding-android-applications.pdf

But I don't think this adds much to this weakness.